src/java.base/share/classes/jdk/internal/math/FloatToDecimal.java

```diff
@@ -32,10 +33,24 @@
 import static java.lang.Math.multiplyHigh;
 import static jdk.internal.math.MathUtils.*;
 
+import sun.nio.cs.ISO_8859_1;
+
 /**
  * This class exposes a method to render a {@code float} as a string.
  */
-public final class FloatToDecimal {
+public final class FloatToDecimal extends ToDecimal {
```

Comment:
The review of this class is the same as for `DoubleToDecimal`, so I won't repeat it here.